android/gradle.properties

```
@@ -1,4 +1,4 @@
 customerio.reactnative.kotlinVersion=1.5.31
 customerio.reactnative.compileSdkVersion=30
 customerio.reactnative.targetSdkVersion=30
-customerio.reactnative.cioSDKVersionAndroid=3.3.0-beta.1
+customerio.reactnative.cioSDKVersionAndroid=3.3.+
```

We are allowing minor updates as well. 

`Major/Breaking.Minor.Fix/Patch`

```suggestion
customerio.reactnative.cioSDKVersionAndroid=[3.3,4.0)
```